diagnostic.md

```diff
@@ -5,74 +5,84 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+The purpose of a backend is to allow data and user info that is used by the client to persist when no sessions are current, and to allow multiple sessions to connect with each other.  It also allows the program to check itself and the requests sent by the user to make sure that they are valid requests and do not compromise the security of the application.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+The controller uses the model to fetch data from the database.
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
+The controller instructs the model how to interact with the database based on the requests that it is given.
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+The rails serializer allows us to limit our interaction with views because of the functionality they provide.
 ```
```

Comment:
More because we build our own Client 'views' with HTML/Handlebars/Javascript etcetera.